data/helpers.d/network

```diff
@@ -0,0 +1,15 @@
+# Check the path syntax
+# Check the slash at the beginning of path and its absence at ending
+#
+# example: ynh_check_path
+#
+# usage: ynh_check_path
+ynh_check_path () {
```

Comment:
IMHO the function should take as argument the variable name. So that we call it like :
`ynh_check_path path` or `ynh_check_path my_app_path`